stock_location_zone/models/stock_location.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2017 Sylvain Van Hoof <svh@sylvainvh.be>
+# Copyright 2018-2019 Jacques-Etienne Baudoux (BCIM sprl) <je@bcim.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+
+
+class StockLocation(models.Model):
+    _inherit = 'stock.location'
+
+    # FIXME: add in selection: shuttle, tray (module vertical lift)
+    kind = fields.Selection([
+        ('zone', 'Picking Zone'),
+        ('area', 'Area'),
+        ('bin', 'Bin'),
```

Comment:
Can't we compute this based on other properties? Example, a leaf is a bin, a record with a `picking_zone_id` is a zone, an area I don't know... Maybe not possible but worth to ask :)

Reply:
BTW the vertical lift ones are computed, so a manual selection as here won't really fit together

Reply:
I would like to be able to use the tray/cell concepts you implemented without having a vertical lift (shuttle). I could have a table or shelf with x/y coordinates that could use the matrix display.

Reply:
I see, makes sense